packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "forceConsistentCasingInFileNames": true,
     "moduleResolution": "NodeNext",
     "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "paths": { "@azure/arm-networkanalytics": ["./src/index"] }
   },
-  "include": ["./src/**/*.ts", "./test/**/*.ts"]
+  "include": ["./src/**/*.ts", "./test/**/*.ts", "samples-dev/**/*.ts"]
```

Comment:
One limitation of tshy is even the "includes" in tsconfig.json contains test/samples-dev but we still can't inlcude them in compile(see [discussion](https://github.com/Azure/azure-sdk-for-js/pull/30810#discussion_r1722344702)). This would be hard for us to detect any generation issue in samples/test.

We plan to have a [command ](https://github.com/Azure/azure-sdk-for-js/pull/30810)in dev-tool to help check in SDK repo. maybe not in recent priority but I feel like we should also include similar check in codegen repo.

@joheredi @wanlwanl @qiaozha how do you think?

Reply:
LGTM